### ISSUE TITLE
deleteFile forceConfirm documentation doesn't reflect functionality

### ISSUE THREAD

**Issue by sghobbs** [FineUploader/fine-uploader#1522]
As of version 5.5.0, documentation for the forceConfirm option for deleteFile reads as follows:

"Bypass prompting the user for confirmation for file deletion, and continue with delete instead."

However, the functionality is implemented to be the exact opposite. The confirmation text will only display if forceConfirm is true. If forceConfirm is false (which it is by default), then the confirmation is overridden.

**Comment by rnicholus** [comment]
The option name declares the intent pretty clearly, but the description is a little hard to follow. I'll push a clarification soon. Thanks for the report.

**Comment by alphestdev** [comment]
"forceConfirm" can be interpreted in both ways: 
 - force the confirmation dialog to appear
 - force the confirmation to Yes/Confirm, skipping the dialog

Just saying.

**Comment by rnicholus** [comment]
Understood. Since I don't plan on renaming the option `forceTheConfirmationDialogToAppear`, this will be addressed with updated documentation, unless you have another preference.

**Comment by rnicholus** [comment]
Fixed in 5.5.2.
